markdown/events/2022/bytesize-gitpod.md

```diff
@@ -25,3 +25,121 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[15:43](https://www.youtube.com/watch?v=kBoC6QBU-M0t=943)
+You can also set all the github settings that you would need to change. Just very quickly, because I know I'm probably running over time, here in the remote browser we can set this to being a live preview. Okay, no you can't... Probably I have done something wrong. There we go! Within this repository we're now making a live view of the actual repository that's changed and then if I wanted to go through the code into the markdown I can actually change the markdown. Like I showed you before you can actually change what's happening in this live preview of the nf-core website. I think this is super useful as well.
+
+[16:30](https://www.youtube.com/watch?v=kBoC6QBU-M0t=990)
+The last thing I wanted to show you was with the github button here, you can look at all the open pull requests you can click on different ones that you're interested in trying to fix. We'll get rid of this for now. It should show you the pull request and give you all the information. If it loads... hopefully it will... Anyway, normally it should come up with all the things you can do on git basically you can do within this Gitpod environment, I think this is really handy. So there we go!
+
+[17:04](https://www.youtube.com/watch?v=kBoC6QBU-M0t=1024)
+You can do everything that you want to do in git, but it's within this github repository and I think this is really helpful. Finally I think I showed everything I needed to show there. I just wanted to mention here... it's only a few slides left... I wanted to mention that also these Gitpod environments are used as extensive training. Sateesh presented really well all the training materials that we have two weeks ago. All of these repositories are now running a Gitpod environment, with all the the code that you need to run them. If you're new to nextflow, this is a really cool way of getting into Gitpod and nextflow, that you can have this amazing environment in the cloud with all the tools that you need.
+
+[17:48](https://www.youtube.com/watch?v=kBoC6QBU-M0t=1068)
+The cost. The free one is really cool, it's only 50 hours per month. You have four parallel workspaces, but you have this 30 minute activity time. As a member of the nf-core github account you can find a free professional open source account. You can get this through the following link. If you click this, there's a form you need to fill in to be able to get a professional account. You can get this through the link. If you click this, there's a professional account for free, which is really nice of Gitpod. For open source developers. Just to say there's not very many threads and I think the maximum size of each repository is about 30 to 50 gigabytes, in case you're testing and you need more than that. But normally the resources are easily enough to do all these different testing things that we want to do.
```

Comment:
```suggestion
The cost. The free one is really cool, it's only 50 hours per month. You have four parallel workspaces, but you have this 30-minute activity time. As a member of the nf-core GitHub account you can find a free professional open-source account. You can get this through the following link. If you click this, there's a form you need to fill in to be able to get a professional account. You can get this through the link. If you click this, there's a professional account for free, which is really nice of Gitpod. For open source developers. Just to say there's not very many threads and I think the maximum size of each repository is about 30 to 50 gigabytes, in case you're testing and you need more than that. But normally the resources are easily enough to do all these different testing things that we want to do.
```